src/locales/en.json

```diff
@@ -9,6 +9,7 @@
     "item_settings": "Settings",
     "item_collect": "Administrative",
     "item_external_drives": "Shared Drives",
```

Comment:
I think this should be `item_shared_drives` to be consistent

Reply:
indeed, seems I missed it